pyproject.toml

```diff
@@ -43,12 +50,14 @@ docs = [
   "furo",
 ]
 tests = [
-  "fsspec[http]>=2022.8.2",
-  "lz4>=0.10",
+  "asdf[all]",
   "psutil",
   "pytest>=8",
   "pytest-remotedata",
 ]
+test = [
```

Comment:
I snuck this in because I often type `pip install asdf[test]` then it complains that `tests` was expected. I figured we could just make both work.

Reply:
I do this too 😅